dpdata/vasp/outcar.py

```diff
@@ -66,19 +76,33 @@ def get_frames (fname, begin = 0, step = 1) :
     cc = 0
     while len(blk) > 0 :
         if cc >= begin and (cc - begin) % step == 0 :
-            coord, cell, energy, force, virial, is_converge = analyze_block(blk, ntot, nelm)
-            if is_converge : 
+            if ml == False:
```

Comment:
This line does not obey [PEP-8](https://peps.python.org/pep-0008/). (Don’t compare boolean values to `True` or `False` using `==`)